### PR TITLE
Email notification for following unfollowing

### LINKED ISSUE #4164: Suggestion: Follow people on Public Lab
Many websites allow users to follow each other and form web. By this people will be able to know what their friends are doing. @ebarry @jywarren @publiclab/mentors what do you suggest?

### PR DESCRIPTION
**Pull request by shubhscoder**
Fixes #4164 

Thanks!

### CODE REVIEW

**Comment by oorjitchowdhary** [comment]
Hi @shubhscoder please show the working of this feature on localhost.. Thanks for your PR

**Comment by shubhscoder** [comment]
localhost_email_working
@oorjitchowdhary I have added the tests and also  screenshot of the working on localhost

**Comment by shubhscoder** [comment]
Good suggestion! We can do this

**Comment by shubhscoder** [comment]
@jywarren do we need some changes here?

File: app/mailers/application_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
+  layout 'mailer'
+end
```
Comment (gauravano) [inline app/mailers/application_mailer.rb]: I think we can remove this

File: app/mailers/relationship_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class RelationshipMailer < ApplicationMailer
```
Comment (gauravano) [inline app/mailers/relationship_mailer.rb]: it would also require change to `ActionMailer::Base`

Comment (gauravano) [reply app/mailers/relationship_mailer.rb]: if we are removing the application helper

**Comment by shubhscoder** [comment]
I will make the required changes @gauravano. Thank you!

**Comment by gauravano** [comment]
Cool!

**Comment by SidharthBansal** [comment]
Hi, just checking if you've gotten stuck on this at all, or if I could help in any way? Thanks!

**Comment by cesswairimu** [comment]
Hello, I am sorry but it seems like this was solved in another PR. Closing this...Let me know if you need help finding another issue. Thanks

**Pull request closed without merging**